packaging/rpm/cri-resource-manager.spec.in

```diff
@@ -5,7 +5,13 @@ Summary: CRI Resource Manager, a CRI proxy with various in-node workload placeme
 License: ASL 2.0 
 URL:     https://github.com/intel/cri-resource-manager
 Source0: https://github.com/intel/cri-resource-manager/archive/cri-resource-manager-__TARVERSION__.tar.bz2
-BuildRequires: go >= 1.12, /usr/bin/make, /usr/bin/install
+BuildRequires: /usr/bin/make, /usr/bin/install
+BuildRequires: /usr/bin/mkdir, /usr/bin/find, /usr/bin/touch, /usr/bin/grep, /usr/bin/sed
+BuildRequires: /usr/bin/tr, /usr/bin/sort, /usr/bin/uniq
+BuildRequires: kernel-devel
```

Comment:
please, clean-up those build-essentials. they should be coming as dependencies to rpm-build.